README.md

```diff
@@ -1,2 +1,40 @@
-# Newsweek_clone_bootstrap
-Microverse Project for learning to use bootstrap. This is a clone of Newsweek clone using this framework.
+## Building with Bootstrap - Newsweek
+
+>This project consists of replicating the <a href="https://www.newsweek.com/" target="_blank">Newsweek</a>💻 website.
+Newsweek is one of many site that uses Bootstrap for its design.
```

Comment:
- [ ] Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.